### DISCUSSION TITLE
Force Mobile view on Desktop

### DISCUSSION THREAD

**Discussion by constantintaivan** [twbs/bootstrap#32059] [Q&A]
Hello,

I'm using the Bootstrap v4.3.1 in my ASP.NET Core project.
Is there a way to force the mobile view on desktop? I need this as a way to keep consistency when testing the app on desktop and provide same look and feel for visitors in some live deployments.
Perhaps a kind of button I can employ to switch on mobile. At best the choice would be stored in browser cookie for next visit.

Thank you,

**Comment by patrickhlauke** [comment]
stick it in an appropriately sized `<iframe>`?

**Reply by constantintaivan** [reply]
Yes, I saw this example: https://stackoverflow.com/questions/44267856/force-desktop-browsers-to-display-mobile-version-of-website
but wanted somehow to avoid it and stick to some bootstrap javascript api if available (a way to apply mobile classes/change media queries triggers)

Same web app runs within a React Native webview, and not sure if would be the right approach. 
As well I don't want to affect viewport meta definition.

**Comment by Priscillarajina14** [comment]
How to do mobile view in js

**Comment by patkon** [comment]
Why don't you just use your browser's developer tools? That's what they're there for (among other things).
